Example/GiniVision/AppCoordinator.swift

```diff
@@ -46,9 +46,36 @@ final class AppCoordinator: Coordinator {
         return giniConfiguration
     }()
     
+    let clientID = "client_id"
+    let clientPassword = "client_password"
+    let clientEmailDomain = "client_email_domain"
+    let credentialsPlistPath = Bundle.main.path(forResource: "Credentials", ofType: "plist")
+    
+    private lazy var client: GiniClient = {[unowned self] in
+        var keys: NSDictionary?
+        if let path = self.credentialsPlistPath,
+            let keys = NSDictionary(contentsOfFile: path),
+            let client_id = keys[self.clientID] as? String,
+            let client_password = keys[self.clientPassword] as? String,
+            let client_email_domain = keys[self.clientEmailDomain] as? String,
+            !client_id.isEmpty, !client_password.isEmpty, !client_email_domain.isEmpty {
+            
+            return GiniClient(clientId: client_id,
+                          clientSecret: client_password,
+                          clientEmailDomain: client_email_domain)
+        }
+        return GiniClient(clientId: "",
```

Comment:
When I did the same thing for Switch, Stephan complain that returning empty strings doesn't make so much sense because it's already pretty obvious that the app wouldn't work. He suggested to throw an exception. I decided to just assert, I think. I would also consider the same thing here.

Reply:
It makes sense for us if we just want to test something not related to analysis results. It could lead to a sad first startup for someone that is testing the Example app, because it is quite likely that they miss adding the `plist` before. 
What do you think?